modules/pubsub/src/main/scala/com/snowplowanalytics/snowplow/streams/pubsub/PubsubFactoryConfig.scala

```diff
@@ -12,9 +12,15 @@ import io.circe.generic.semiauto._
 
 /**
  * Configures the Pubsub Factory
+ *
+ * @param gcpUserAgent
+ *   configures the user-agent header in requests to pubsub
+ * @param emulatorHost
+ *   Optional, e.g. `localhost:8181`. Use the pubsub emulator for local development.
  */
 case class PubsubFactoryConfig(
-  gcpUserAgent: GcpUserAgent
+  gcpUserAgent: GcpUserAgent,
+  emulatorHost: Option[String]
```

Comment:
I wonder, should each app be in charge of checking if we are using an emulator or not, shouldn't we rather let this lib figure it out?

Reply:
Yeah sorry this was me being focused on the use of `PUBSUB_EMULATOR_HOST` like [here](https://cloud.google.com/pubsub/docs/emulator#manually_setting_the_variables), but we don't need to do that.